includes/admin.inc

```diff
@@ -1278,6 +1359,16 @@ function _islandora_solr_admin_settings_validate($form, &$form_state) {
   $limits = explode(',', $form_state['values']['islandora_solr_num_of_results_advanced']);
   $form_state['values']['islandora_solr_num_of_results_advanced'] = array_filter(array_map('trim', $limits), 'is_numeric');
   $form_state['values']['islandora_solr_namespace_restriction'] = preg_replace('/:$/', '', $form_state['values']['islandora_solr_namespace_restriction']);
+
+  if (preg_match('/^\d*$/', $form_state['values']['islandora_solr_query_phrase_slop']) !== 1) {
```

Comment:
Doesn't this regex match a zero-length string? Or is that something you want?

Reply:
That is intended behaviour: the query phrase slop can either be empty or a natural number.